pyproject.toml

```diff
@@ -6,17 +6,16 @@ authors = ["Team Microdata"]
 license = "Apache"
 
 [tool.poetry.dependencies]
-python = "^3.12"
+python = "^3.13"
 fastapi = "^0.115.2"
-pyarrow = "^15.0.0"
+pyarrow = "^20.0.0"
 uvicorn = {version = "^0.34.2", extras = ["h11==0.16.0"]}
-pandas = "^2.2.0"
-numpy = "1.26.4"
 python-dotenv = "^1.0.1"
 PyJWT = "^2.4.0"
 aiofiles = "^24.0.0"
 cryptography = "^44.0.1"
 PyYAML = "^6.0"
+pandas = "^2.2.3"
```

Comment:
move to dev deps?

Reply:
not sure if we use any of the transitive dependencies in prod. Will mark this for later and try this upgrade first 👍🏻